src/node/rpc/member_frontend.h

```diff
@@ -946,6 +946,24 @@ namespace ccf
         .set_openapi_hidden(true)
         .install();
 
+      using JWTKeyMap = std::map<JwtKeyId, Cert>;
+
+      auto get_jwt_keys = [this](auto& ctx, nlohmann::json&& body) {
+        auto keys = ctx.tx.template ro<JwtPublicSigningKeys>(
+          ccf::Tables::JWT_PUBLIC_SIGNING_KEYS);
+        JWTKeyMap kmap;
+        keys->foreach([&kmap](const auto& kid, const auto& kpem) {
+          kmap[kid] = kpem;
```

Comment:
Could we bundle the issuer here as well, by looking up the `public:ccf.gov.jwt.public_signing_key_issuer` table?

Reply:
We could, but we won't until there's a use case for it.